packages/mui-material/src/index.js

```diff
@@ -405,3 +403,10 @@ export { default as GlobalStyles } from './GlobalStyles';
 export * from './GlobalStyles';
 
 export { StyledEngineProvider } from './styles';
+
+export { default as unstable_composeClasses } from '@mui/base/composeClasses';
```

Comment:
Don't we need to have `export * from '@mui/base/composeClasses';` and `export * from '@mui/base/generateUtilityClasses';` as well?